.gitignore

```diff
@@ -11,4 +11,5 @@ test/acs-engine-test/acs-engine-test
 
 test/acs-engine-test/acs-engine-test.exe
 pkg/operations/junit.xml
-pkg/operations/kubernetesupgrade/junit.xml
\ No newline at end of file
+pkg/operations/kubernetesupgrade/junit.xml
+pkg/acsengine/templates.go
```

Comment:
We cannot ignore templates.go file. 

If we keep a legacy template.go file for ever, new changes into parts/ won't be taken, right?